src/test/java/org/openstreetmap/atlas/checks/validation/linear/edges/SignPostCheckTestRule.java

```diff
@@ -396,6 +396,93 @@
                                     "destination=somewhere" }) })
     private Atlas motorwayMotorwayLinkWithJunctionAndDestinationAtlas;
 
+    @TestAtlas(
+            // nodes
+            nodes = { @Node(coordinates = @Loc(value = HIGHWAY_1)),
+                    @Node(coordinates = @Loc(value = HIGHWAY_3), tags = {
+                            "highway=motorway_junction" }),
+                    @Node(coordinates = @Loc(value = HIGHWAY_5)),
+                    @Node(coordinates = @Loc(value = LINK_1)),
+                    @Node(coordinates = @Loc(value = LINK_4)) },
+            // edges
+            edges = {
+                    @Edge(coordinates = { @Loc(value = HIGHWAY_1), @Loc(value = HIGHWAY_2),
```

Comment:
Not necessarily a comment about this line, but can you add a test that includes two way edges and make sure everything is as intended? Not sure of your exposure with the test atlas annotation, but it is pretty raw. To have a properly formatted reverse edge you need to create another ```@Edge``` with the locations reversed. And I also believe that it needs to have a flipped ID compared to the main edge. For example  the masterEdge ```@Edge( id = x ...``` and the reverse  ```@Edge( id = -x ...```.

Another thing to look into is 
```
@TestAtlas (loadFromJosmOsmResource = "relativePathFromSamePackagePathInTestResources")
private testAtlas;
```
This allows you to open JOSM and draw the problem in a layer and save that layer as an osm xml file to the Test resources package path, then use that data as a test atlas. It will handled edges/reverse edges provided you tag them correctly.

Reply:
I was attempting to test that in `motorwayMotorwayLinkTwoWayBranchMissingDestinationAtlas`, but did not quite have it all right. I have updated that test to use ids and have more 2 way edges.